lib/gman_client.rb

```diff
@@ -8,4 +8,14 @@ def self.drivers
     respond = request.api.v1.drivers.get
     respond.map(&:to_h)
   end
+  def self.driver_commission_histories
+    request = Blanket.wrap("http://localhost:3000", :extension => :json)
```

Comment:
I would recommend replacing the static strings with something from configuration, ENV or Figaro for example.